docs/User-Guide_Advanced-Configuration.md

```diff
@@ -0,0 +1,406 @@
+
+First, find the matching HDMI output (the `x11-xserver-utils` package must be installed):
+
+```sh
+xrandr --listmonitors
+```
+
+Then, calculate the VESA CVT mode line. The following command does this for the a custom resolution of 1440x900.
+
+```sh
+cvt 1440 900
+```
+
+The command will output a new modeline. For our example, it may look like this:
+
+```
+# 1440x900 59.89 Hz (CVT 1.30MA) hsync: 55.93 kHz; pclk: 106.50 MHz
+Modeline "1440x900_60.00"  106.50  1440 1528 1672 1904  900 903 909 934 -hsync +vsync )
+```
+
+The new modeline can then be used directly to create and add the new mode, and enable it. The following commands will do that for the output device _HDMI-1_.
+
+```sh
+xrandr --newmode "1440x900_60.00" 106.50 1440 1528 1672 1904 900 903 909 934 -hsync +vsync
+xrandr --addmode HDMI-1 1440x900_60.00
+xrandr --output HDMI-1 --mode 1440x900_60.00
+```
+
+If it works well, the new mode can also be added to X.Org's configuration in `/etc/X11/xorg.conf.d/` to make it permanently available/active. Otherwise, these commands will have to be executed after every reboot. To load this resolution automatically after starting the device, add the following section to e.g. `/etc/X11/xorg.conf.d/40-monitor.conf` (create the file if it does not exist):
+
+```
+Section "Monitor"
+	Identifier "HDMI-1"
+	Modeline "1440x900_60.00" 106.50 1440 1528 1672 1904 900 903 909 934 -hsync +vsync
+	Option "PreferredMode" "1440x900"
+EndSection
+```
+
+After a restart, the graphical session should automatically be shown in the chosen resolution.
+
+
+## Alter the CPU frequency
+
+Some boards allow to adjust the CPU speed by editing the file `/etc/default/cpufrequtils` and alter the  **min_speed** and/or **max_speed** variable. Changing these values require restarting `cpufrequtils.service` to activate the new settings:
+
+```sh
+systemctl restart cpufrequtils.service
+```
+
+
+## Swap for experts
+
+By default, Armbian implements ZRAM (writing nothing to 'disk', but compressing memory pages in RAM). In case you often run into out-of-memory (OOM) errors and your device has some capable storage (e.g. a securely attached NVMe or SATA SSD), you might want to use ZSWAP instead.
+
+Check whether your kernel has zswap enabled. If yes, the following command
+
+```sh
+dmesg | grep zswap
+```
+
+should return some output. If that is the case, create a swapfile or a swap partition the traditional way: edit `/etc/default/armbian-zram-config` so that it reads `SWAP=false`. Reboot, and you're done.
+
+Zswap performs a lot better than the combination of ZRAM and 'swap on disk' in parallel.
+
+
+## Switch kernels
```

Comment:
I would propse to drop this from here as its a part of armbian-confg - or link there.